lib/handlers/quote/quote.ts

```diff
@@ -258,6 +262,7 @@ export class QuoteHandler extends APIGLambdaHandler<
       gasPriceWei,
       methodParameters,
       blockNumber,
+      simulationError,
     } = swapRoute
 
     const routeResponse: Array<(V3PoolInRoute | V2PoolInRoute)[]> = []
```

Comment:
Lets emit some metrics on this so we can track usage. One for simulation requested, one for success and one for failure. See L86 for an example